about.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;1,300&display=swap" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Fashion Conference</title>
+</head>
+<body>
+    <header>
+        <div class="upper-bar">
+            <ul>
+                <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My page</a></li>
+                <li><a href="#">Logout</a></li>
+                
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-menu">
+                <a href="/index.html">
```

Comment:
- [ ] Currently, this link takes to a 404 page, please fix this 👍
- [ ] Also, in the mobile menu, it doesn't seem to be working at all, please fix this issue in order for the user to navigate easily between the website pages. 🙏

Reply:
- [ ] @MahmoudBakr23 Thanks for the observation. I have rectified it. Apparently, I didn't specify the directory well.